Source/Core/Core/IOS/IPC.cpp

```diff
@@ -348,6 +391,11 @@ constexpr std::array<IosMemoryValues, 31> ios_memory_values = {
          0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
          RAM_VENDOR, 0x93600000, 0x93620000,       0,
      },
+     {60,         0x3c181e,   0x112408,         MEM1_SIZE,
```

Comment:
Does lint format it like that, or did you simply miss a newline here between the brace and the IOS number?